10-Feb-2022 00:48:29.191 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
10-Feb-2022 00:48:29.191 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
10-Feb-2022 00:48:29.191 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@6974a715')
10-Feb-2022 01:16:47.590 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
10-Feb-2022 01:16:47.590 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
10-Feb-2022 01:27:05.381 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
10-Feb-2022 01:27:05.381 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
10-Feb-2022 01:27:05.383 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@d59970a')
10-Feb-2022 02:31:25.309 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
10-Feb-2022 02:31:25.309 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
10-Feb-2022 18:32:19.757 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
10-Feb-2022 18:32:19.757 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
10-Feb-2022 18:32:19.759 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@22d6cac2')
10-Feb-2022 18:38:10.240 SEVERE [http-nio-8080-exec-8] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [/redecor] threw exception [An exception occurred processing [login.jsp] at line [14]

11:     }
12: 
13: Class.forName("org.mariadb.jdbc.Driver");
14: java.sql.Connection con = DriverManager.getConnection("jdbc:mariadb://localhost:3306/userinfo", "root", "");
15: Statement st = con.createStatement();
16: ResultSet rs = st.executeQuery("select userid,password from users");
17: try {


Stacktrace:] with root cause
	java.net.ConnectException: Connection refused: connect
		at java.base/sun.nio.ch.Net.connect0(Native Method)
		at java.base/sun.nio.ch.Net.connect(Net.java:574)
		at java.base/sun.nio.ch.Net.connect(Net.java:563)
		at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:588)
		at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:333)
		at java.base/java.net.Socket.connect(Socket.java:648)
		at java.base/java.net.Socket.connect(Socket.java:597)
		at org.mariadb.jdbc.internal.mysql.MySQLProtocol.connect(MySQLProtocol.java:288)
		at org.mariadb.jdbc.internal.mysql.MySQLProtocol.connectWithoutProxy(MySQLProtocol.java:624)
		at org.mariadb.jdbc.internal.common.Utils.retrieveProxy(Utils.java:541)
		at org.mariadb.jdbc.Driver.connect(Driver.java:101)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
		at org.apache.jsp.login_jsp._jspService(login_jsp.java:132)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:228)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:893)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1723)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
10-Feb-2022 18:39:24.066 SEVERE [http-nio-8080-exec-2] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [/redecor] threw exception [An exception occurred processing [login.jsp] at line [14]

11:     }
12: 
13: Class.forName("org.mariadb.jdbc.Driver");
14: java.sql.Connection con = DriverManager.getConnection("jdbc:mariadb://localhost:3306/userinfo", "root", "");
15: Statement st = con.createStatement();
16: ResultSet rs = st.executeQuery("select userid,password from users");
17: try {


Stacktrace:] with root cause
	java.net.ConnectException: Connection refused: connect
		at java.base/sun.nio.ch.Net.connect0(Native Method)
		at java.base/sun.nio.ch.Net.connect(Net.java:574)
		at java.base/sun.nio.ch.Net.connect(Net.java:563)
		at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:588)
		at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:333)
		at java.base/java.net.Socket.connect(Socket.java:648)
		at java.base/java.net.Socket.connect(Socket.java:597)
		at org.mariadb.jdbc.internal.mysql.MySQLProtocol.connect(MySQLProtocol.java:288)
		at org.mariadb.jdbc.internal.mysql.MySQLProtocol.connectWithoutProxy(MySQLProtocol.java:624)
		at org.mariadb.jdbc.internal.common.Utils.retrieveProxy(Utils.java:541)
		at org.mariadb.jdbc.Driver.connect(Driver.java:101)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
		at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
		at org.apache.jsp.login_jsp._jspService(login_jsp.java:132)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:228)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:893)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1723)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
10-Feb-2022 19:12:21.092 SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [jsp] in context with path [/redecor] threw exception [Unable to compile class for JSP: 

An error occurred at line: [317] in the generated java file: [J:\ide programs\apache-tomcat-9.0.50\work\Catalina\localhost\redecor\org\apache\jsp\product_005fview_jsp.java]
Syntax error, insert "}" to complete Block

An error occurred at line: [334] in the generated java file: [J:\ide programs\apache-tomcat-9.0.50\work\Catalina\localhost\redecor\org\apache\jsp\product_005fview_jsp.java]
Syntax error on token "}", delete this token

An error occurred at line: [335] in the generated java file: [J:\ide programs\apache-tomcat-9.0.50\work\Catalina\localhost\redecor\org\apache\jsp\product_005fview_jsp.java]
Syntax error, insert "}" to complete ClassBody

Stacktrace:] with root cause
	org.apache.jasper.JasperException: Unable to compile class for JSP: 

An error occurred at line: [317] in the generated java file: [J:\ide programs\apache-tomcat-9.0.50\work\Catalina\localhost\redecor\org\apache\jsp\product_005fview_jsp.java]
Syntax error, insert "}" to complete Block

An error occurred at line: [334] in the generated java file: [J:\ide programs\apache-tomcat-9.0.50\work\Catalina\localhost\redecor\org\apache\jsp\product_005fview_jsp.java]
Syntax error on token "}", delete this token

An error occurred at line: [335] in the generated java file: [J:\ide programs\apache-tomcat-9.0.50\work\Catalina\localhost\redecor\org\apache\jsp\product_005fview_jsp.java]
Syntax error, insert "}" to complete ClassBody

Stacktrace:
		at org.apache.jasper.compiler.DefaultErrorHandler.javacError(DefaultErrorHandler.java:102)
		at org.apache.jasper.compiler.ErrorDispatcher.javacError(ErrorDispatcher.java:213)
		at org.apache.jasper.compiler.JDTCompiler.generateClass(JDTCompiler.java:487)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:397)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:367)
		at org.apache.jasper.compiler.Compiler.compile(Compiler.java:351)
		at org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:605)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:399)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:379)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:327)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:228)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:893)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1723)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
